### DISCUSSION TITLE
dell u2419hc and dell s2721dgf

### DISCUSSION THREAD

**Discussion by plsdice** [MonitorControl/MonitorControl#1335] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

hi there.

I run this app connected to the monitors above. the brightness control works fine however the volume control doesnt seem to affect the plugged in devices. 

I have experimented with both hdmi and thunderbolt 3 for this 

### Expected behavior

I would expect the device connected over 3.5mm to either of these devices to reduce its volume however this does not happen. the ui appears but the volume maintains. 

### Environment Information (please complete the following information)

```markdown
- macOS version: 13.1
- Mac model: M1 Pro 14
- MonitorControl version: 4.1.0
- Monitor(s): u2419hc.  s2721dgf
- Monitor Cable(s)/Connection(s): usbc and hdmi
- Apple Silicon/M1 (yes or no): yes M1 Pro
```

**Comment by waydabber** [comment]
You are either using HDMI which is not supported in MonitorControl (you can use BetterDisplay for that) or the display does not support DDC or if hardware brightness works then it does not support DDC volume.
